_rules/links-with-identical-names-serve-equivalent-purpose-b20e66.md

```diff
@@ -282,18 +646,98 @@ Links created via scripting, but without the semantic role of link:
 
 #### Inapplicable Example 5
 
-Links do not have [accessible names][accessible name]:
+These links do not have [accessible names][accessible name].
 
 ```html
 <a href="http://facebook.com"></a> <a href="http://twitter.com"></a>
 ```
 
 #### Inapplicable Example 6
 
-Image links do not have [accessible names][accessible name]:
+These image links do not have [accessible names][accessible name].
 
 ```html
 <a href="http://facebook.com"><img src="facebook.jpg"/></a> <a href="http://twitter.com"><img src="twitter.jpg"/></a>
 ```
 
+#### Inapplicable Example 7
```

Comment:
As above, this rule isn't about shadow DOM. inapplicable example 7 and 8 should be removed.